Revert/Sources/AlertViewController.swift

```diff
@@ -49,10 +49,8 @@ extension AlertViewController {
     case .ActionSheet:
```

Comment:
After discussing more we've decided to keep the UIAlertView vs UIAlertController example to show support for those legacy controls.